crypto/lhash/lhash.c

```diff
@@ -97,14 +97,14 @@ void *OPENSSL_LH_insert(OPENSSL_LHASH *lh, void *data)
         (*rn)->data = data;
         lh->num_replace++;
     }
-    return (ret);
+    return (void *)(ret);
```

Comment:
I know the original had it, but since you're modifying this line, please remove the extra parens around ret.  Here and in the other instances, below.